.projenrc.js

```diff
@@ -29,6 +29,7 @@ const project = new awscdk.AwsCdkConstructLibrary({
   releaseTagPrefix: `awscli-v${MAJOR_VERSION}`,
   releaseWorkflowName: releaseWorkflowName,
   defaultReleaseBranch: defaultReleaseBranchName,
+  deps: ['@aws-cdk/interfaces'],
```

Comment:
Can be a devDep

Reply:
But we need to expose an `ILambdaLayerAsset` in this packages API, so that makes it a direct dep, right?

Reply:
Hmm for the purposes of jsii, I think you are right, yes. Never mind then.

Reply:
I temporarily removed this until this PR https://github.com/aws/aws-cdk/pull/22538 Is released anyway.